controllers/doguExportManager.go

```diff
@@ -0,0 +1,147 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudogu/k8s-dogu-operator/v3/api/ecoSystem"
+	"github.com/cloudogu/k8s-dogu-operator/v3/controllers/resource"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"time"
+
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	k8sv2 "github.com/cloudogu/k8s-dogu-operator/v3/api/v2"
+)
+
+const (
+	ChangeExportModeEventReason        = "ChangeExportMode"
```

Comment:
I hate to be that guy but I actually DO read doc comments in my IDE ^^

Exported symbols need a doc comment.

![Toy story meme: Exported symbols. Exported symbols everywhere](https://github.com/user-attachments/assets/21106f35-637d-4b91-bc5e-f3482421f4e4)